mapviz_plugins/src/pointcloud2_plugin.cpp

```diff
@@ -333,10 +349,11 @@ namespace mapviz_plugins
 
   void PointCloud2Plugin::BufferSizeChanged(int value)
   {
-    buffer_size_ = value;
+    buffer_size_ = (size_t)value;
```

Comment:
I don't see an above comment?

But I'm guessing you commented that I was lazy for using a C-style cast, and you are correct, it just slipped my mind that I should really use `static_cast` instead.